app/javascript/components/TenantView.jsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {Avatar, Button, Tag} from 'antd';
+import 'antd/dist/antd.css';
+import moment from 'moment';
```

Comment:
seems like this isn't used

Reply:
Oh, this file shouldn't be in this PR. Will remove.